canonical-websites/tutorials.ubuntu.com
# Update polymer and dependencies

## Please review and let me merge, as I will update the build job

This:
- Updates Polymer and uses the polymer-cli through a locally installed NPM version.
- Prepares yarn scripts, which are compatible with `./run` command
- Sorts and removes unused dependencies
- Removes reference to `app-logo` which does not exist.

## QA
- `yarn install`
- `bower install`
- `Yarn run serve`
- Open web console and check for errors through site
- The Tutorials logo should remain the same
